Documentation/sourcebuild-in-repos/README.md

```diff
@@ -0,0 +1,10 @@
+# Repo owner's handbook for source-build
+
+These docs aim to be a collection of useful information for repo owners who work with and participate in source-build.
+
+## Index
+
+[Adding new dependencies](new-dependencies.md)
+[Updating dependencies](update-dependencies.md)
+[Considerations when adding features](adding-features.md)
+[General notes on the source-build build](build-info.md)
```

Comment:
These could probably be a bulleted list so they don't appear on the same line in the rich markdown view.

Reply:
Fixed